code/game/mecha/mecha.dm

```diff
@@ -502,7 +504,7 @@
 				check_for_internal_damage(list(MECHA_INT_TEMP_CONTROL,MECHA_INT_TANK_BREACH,MECHA_INT_CONTROL_LOST))
 			else
 				check_for_internal_damage(list(MECHA_INT_FIRE,MECHA_INT_TEMP_CONTROL,MECHA_INT_TANK_BREACH,MECHA_INT_CONTROL_LOST,MECHA_INT_SHORT_CIRCUIT))
-		if(. >= 5 || prob(33))
+		if((. != 1 && leg_overload_mode) && (. >= 5 || prob(33))) //If it takes 1 damage and leg_overload_mode is true, do not say TAKING DAMAGE! to the user several times a second.
```

Comment:
This won't work as intended currently. 
For example, given `. = 10` and `leg_overload_mod = FALSE`, we'll get:
```dm
if((TRUE && FALSE) && TRUE)
```
Which of course equates to:
```dm
if(FALSE && TRUE)
```
Which means that having leg overload mode off will prevent valid messages from appearing.